public_html/wp-content/plugins/wcpt/wcpt-pilot-event/pilot-event-loader.php

```diff
@@ -0,0 +1,477 @@
+<?php
+
+// define( 'WCPT_POST_TYPE_ID',   'wordcamp'           );
+// define( 'WCPT_YEAR_ID',        'wordcamp_year'      );
+// define( 'WCPT_PILOT_EVENT_SLUG',           'pilot-events'          );
```

Comment:
Or this one.

Reply:
IMO it'd be better to use `WCPT_PILOT_EVENT_SLUG` than `WCPT_POST_TYPE_ID`, to avoid confusion between the two post types

Reply:
Noting, we'll need to go through all the plugins and replace the direct references to `WCPT_POST_TYPE_ID` and call [this function](https://github.com/WordPress/wordcamp.org/pull/937/files#diff-790ef1c27b93f13a8bb852756e3352013fb78410c737c8f361515f0ae5663dcaR22).